data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/AuditService.java

```diff
@@ -15,40 +15,43 @@
  */
 package uk.gov.gchq.palisade.service.data.service;
 
-import feign.Response;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import uk.gov.gchq.palisade.service.Service;
 import uk.gov.gchq.palisade.service.data.request.AuditRequest;
 import uk.gov.gchq.palisade.service.data.web.AuditClient;
 
-import java.net.URI;
-import java.util.concurrent.Executor;
-import java.util.function.Supplier;
-
+/**
+ * Wraps a feign client to provide an implementation of {@link Service}
+ */
```

Comment:
Is this true  still - I see no mention of a feign client in the definition.
Suggest something along the lines of implements AuditService providing an implementation of {@link Service}

Reply:
updated